loops_lab.js

```diff
@@ -0,0 +1,195 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+console.log('1.--------------');
+console.log('While Loop');
+
+let num = 15;
+let startingNum = num;
+while (num > 1 && num) {
+  if (num === startingNum) {
+    num--;
+  }
+
+  console.log(num);
+  num--;
+}
+
+console.log('For Loop');
+
+for (num = 8; num > 1; --num) {
```

Comment:
Also it's good practice to declare variable inside for loops using "let". For example, for(let num = 8; num > 1; --num). It is helpful to avoid case like this 

let num = "lalala"
for (num = 8; num > 1; --num) {
  if (num === 8) {
    num--;
  }
  console.log(num);
}

console.log("what is num?", num) <== num will become 1 instead of "lalala"